website/views/organization.py

```
@@ -40,6 +40,13 @@
 from rest_framework import status
 from rest_framework.authtoken.models import Token
 
+from slack_sdk.web import WebClient
+from slack_sdk.errors import SlackApiError
+from django.shortcuts import render
```

Ruff (0.14.4)

Redefinition of unused `render` from line 30

Remove definition: `render`

(F811)